scripts/common/upgrade.sh

```diff
@@ -315,3 +319,12 @@ function upgrade_maybe_remove_kubeadm_network_plugin_flag() {
     fi
     sed -i 's/ \?--network-plugin \?[^ "]*//' /var/lib/kubelet/kubeadm-flags.env
 }
+
+# delete the flannel pod on the node so that CNI plugin binaries are recreated
```

Comment:
```suggestion
# delete the flannel pod on the node so that CNI plugin binaries are recreated
# workaround for https://github.com/kubernetes/kubernetes/issues/115629
```